wrappers/python/performance/llama_default_config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "activation_function": "silu",
+    "architectures": [
+      "LlamaCasualForLM"
+    ],
+    "attention_dropout": 0.0,
+    "initializer_range": 0.02,
+    "rms_norm_eps": 1e-06,
+    "model_type": "llama",
+    "flashattention": false,
+    "redux": false,
+    "dtype": "fp32",
+    "eos_token_id": 2,
+    "bos_token_id": 1,
+    "attention_bias": false,
+    "rope_theta": 10000,
+    "mlp_bias": false,
+    "vocab_size": 32000,
+    "hidden_size": 4096,
```

Comment:
What is the "default" size? It has only 8 layers, but hidden size is 4096. Model 1.3B has 24 layers with 2048 embedding size.